includes/blast_ui.admin.inc

```diff
@@ -156,6 +164,100 @@ function blast_ui_admin_form_validate($form, &$form_state) {
 
 }
 
+
+/*
+**
+** custom method disabled the web services 
+**
+*/
+function disable_web_services()
+{
+    $ar=array('services', 'rest_server');
+    $result = module_disable($ar,true);
+    try{
+    $eid = 1;
+    $num_deleted = db_delete('services_endpoint')->condition('eid', $eid)->execute();  
```

Comment:
I have made changes on suggestion. I have taken care of this issue like you have suggested precisely here.